### ISSUE TITLE
chown error?

### ISSUE THREAD

**Issue by NoLooseEnds** [binhex/arch-rtorrentvpn#57]
Hi, I'm pretty new to this:

I mounted two a NFS paths: 
- /mnt/host/download mounted to /data
- /mnt/host/docker mounted to /config

I ran the docker container, and I got a log file in /mnt/host/docker (i.e /config) going through every file/folder in the /mnt/host/download directory and trying to chown (permission denied).

This is my docker run code:
```		
docker run -d \
		    --cap-add=NET_ADMIN \
		    -p 9080:9080 \
		    -p 9443:9443 \
		    -p 8118:8118 \
		    -p 3000:3000 \
		    --name=rtorrentvpn \
		    -v /mnt/<host>/download:/data \
		    -v /mnt/<host>/docker/:/config \
		    -v /etc/localtime:/etc/localtime:ro \
		    -e VPN_ENABLED=yes \
		    -e VPN_USER=<user> \
		    -e VPN_PASS=<pass> \
		    -e VPN_PROV=custom \
		    -e VPN_OPTIONS= \
		    -e STRICT_PORT_FORWARD=yes \
		    -e ENABLE_PRIVOXY=yes \
		    -e ENABLE_FLOOD=yes \
		    -e ENABLE_AUTODL_IRSSI=yes \
		    -e LAN_NETWORK=10.0.0.0/8 \
		    -e NAME_SERVERS=194.187.251.6,38.132.106.139,1.1.1.1,1.0.0.1 \
		    -e DEBUG=true \
		    -e PHP_TZ=CEST \
		    -e UMASK=000 \
		    -e PUID=0 \
		    -e PGID=0 \
		    binhex/arch-rtorrentvpn
```

Why is it trying to change the owner of my files?

Thanks.

**Comment by binhex** [comment]
'''I ran the docker container, and I got a log file in /mnt/host/docker (i.e /config) going through every file/folder in the /mnt/host/download directory and trying to chown (permission denied).'''

you sure about it changing permissions to your download folder?, it will chown for folders inside the docker containers file system and it will chown for /config/openvpn. it will do this once only, there is no reference to chown for /data, relevant lines of code can he seen here, search for 'chown':-

https://github.com/binhex/arch-rtorrentvpn/blob/master/build/root/install.sh

line 201 and line 246

FYI the chown is required due to the fact you can define which user and group you want the container to run as by configuring the PUID and PGID values

**Comment by NoLooseEnds** [comment]
Yes, sure. I don't have the log file anymore, but it was flooded with paths for my files inside download, with the "chown" command and permission denied. So it did not change the owner, but tried to.

I'm going to experiment a bit more with this. I'll try to set the UID GID to my user, but I would think 0 (ie. root) should work.

**Comment by binhex** [comment]
setting puid and pgid to 0 does work, and will set it to user root group root, but the choice is yours as to what you run the container as, just make sure you delete /config/perms.txt if you start changing it, otherwise permissions will not be set correctly and you may get permissions issues.
